include/TrustWalletCore/TWStoredKey.h

```diff
@@ -78,6 +114,16 @@ struct TWStoredKey* _Nonnull TWStoredKeyCreateLevel(TWString* _Nonnull name, TWD
 /// \return The stored key as a non-null pointer
 TW_EXPORT_STATIC_METHOD struct TWStoredKey* _Nonnull TWStoredKeyCreate(TWString* _Nonnull name, TWData* _Nonnull password);
 
+/// Creates a new key.
+///
+/// \deprecated use TWStoredKeyCreateLevel.
```

Comment:
maybe we don't need this method? add a deprecated mark to `TWStoredKeyCreateLevel` and always use `TWStoredKeyCreateLevelAndEncryption` in the future